### PR TITLE
Updated readme and scan results for Metadata Store v1.1.1

### PR DESCRIPTION
**Pull request by Pivotal-Jeff-Jun**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -52,8 +68,40 @@ This release has the following security issues:
...
+- **Missing persistent volume:**
+  After Supply Chain Security Tools - Store is deployed, `metadata-store-db` Pod might fail for missing
+  volume while `postgres-db-pv-claim` pvc is in the `PENDING` state.
+  This issue may occur if the cluster where Supply Chain Security Tools - Store is deployed does not have
+  `storageclass` defined.
+
+  The provisioner of `storageclass` is responsible for creating the persistent volume after
+  `metadata-store-db` attaches `postgres-db-pv-claim`. To resolve this issue, see
+  [Missing Persistent Volume](troubleshooting.html#missing-persistent-volume)
+  in _Troubleshooting Tanzu Application Platform_.
```
Comment (xtreme-debbie-chen) [inline release-notes.md:99]: I'm assuming this was copied from the somewhere else, but the newlines in this whole known issues section is a little strange... Was that intentional or a bug?

**Review by Manifaust — APPROVED** [review]

**Review by xtreme-debbie-chen — APPROVED** [review]

File: scst-store/sca_scans/sca-scanning-results.md [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+
+#### <a id='api-backend-cr'></a>API Backend Code Repository
+
+* Date: March 22, 2022
+* Results: No known vulnerabilities
+
+#### <a id='cli-cr'></a>CLI Repository
+
+* Date: March 22, 2022
+* Results: No critical, and 2 high vulnerabilities: [CVE-2015-5237](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5237), [CVE-2021-22570](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22570). No medium or low vulnerabilities. Multiple medium and low vulnerabilities. For more information, see the [CLI Repository v1.1.1 CycloneDX file content](cli-repository-v1.1.1-grype-result.md).
```
Comment (xtreme-jason-smith) [inline scst-store/sca_scans/sca-scanning-results.md:33]: I think you forgot to remove the `No medium or low vulnerabilities.` phrase here. The next phrase says that there are some.

File: scst-store/sca_scans/sca-scanning-results.md [hunk 3]
```diff
@@ -1,4 +1,53 @@
...
+
+#### <a id='kube-rbac-proxy-ci'></a>Kube RBAC Proxy Container Image
+
+* Date: March 22, 2022
+* Results: No critical, and 3 high vulnerabilities: [CVE-2022-21698](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21698), [CVE-2015-5237](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5237), [CVE-2021-22570](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22570). No medium or low vulnerabilities. Multiple medium and low vulnerabilities. For more information, see the [Kube RBAC Proxy Container Image v1.1.1 CycloneDX file content](kube-rbac-proxy-container-image-v1.1.1-grype-result.md).
```
Comment (xtreme-jason-smith) [inline scst-store/sca_scans/sca-scanning-results.md:38]: Same here, forgot to remove `No medium or low vulnerabilities.`

File: release-notes.md (hunk 1 above)
Comment (Pivotal-Jeff-Jun) [reply release-notes.md:99]: Despite the new lines here, in the markdown output view it's single line. This was edited by docs team. I think they did this to not have these line too long.

**Pull request merged**